### ISSUE TITLE
Lock & unlock tag ?

### ISSUE THREAD

**Issue by spljs** [gotthardp/python-mercuryapi#130]
Hello,

I'm wondering if this library has some lock/unlock gen2 tag capabilities with a password. Same for write protection
I don't see any implementation for that

Thanks

**Comment by lordmundi** [comment]
I agree. This would be a great addition to support the passwords capability in the mercury-api

**Comment by pytremb** [comment]
I now have a requirement to modify GEN2 tags that have been assigned an Access Password in the Reserved Memory Bank. Any idea if this great Python wrapper will be modified to support passwords? Thanks.
